src_assets/linux/misc/postinst

```diff
@@ -1,17 +1,5 @@
 #!/bin/sh
 
-export GROUP_INPUT=input
-
-if [ -f /etc/group ]; then
-        if ! grep -q $GROUP_INPUT /etc/group; then
-                echo "Creating group $GROUP_INPUT"
-
-                groupadd $GROUP_INPUT
```

Comment:
I think this broke the Docker builds.

Reply:
Yes, you're right. I removed creation of `input` group from DEB/RPM postinstall script since udev rule makes it unnecessary. However, Docker images still try to create a `lizard` user and add it to `input` group:
```bash
useradd -lm -d ${HOME} -s /bin/bash -g "${PGID}" -G input -u "${PUID}" "${UNAME}"
```
Unfortunately, Debian/Ubuntu images don't have `input` group by default. That's why the build fails.

In this case I suggest to create `lizard` user without `input` group. Normally, users created in Docker containers don't have access to host devices. But as I see, UID and GID of `lizard` user are set to match UID and GID of host user. This means that if we properly install udev rule on the host, both host user and `lizard` user should be able to access Sunshine virtual devices. No `input` group needed.

Applied the changes in the commit below.

Reply:
@ABeltramo does this seem like a sane approach to you?

Reply:
This is a tricky one and would have to be tested, but if the udev rules are going to give group access to the specified GID and that's the same group mapped to the container, it should work. 
Am I getting the approach here right?

Reply:
@ABeltramo Basically yes, except that udev rules are going to give **user** access to the specified UID and that's the same user mapped to the container. Groups are not involved at all. The whole point of this PR is to avoid group permissions and rely on user permissions exclusively. It is intended to work in the following way:

1. We install udev rule on the host.
2. udev rule sets up ACL on `/dev/uinput` to grant R/W access to user on the host. Here is how permissions look on my system:
```
$ getfacl /dev/uinput
getfacl: Removing leading '/' from absolute path names
# file: dev/uinput
# owner: root
# group: input
user::rw-
user:kodemeister:rw-
group::rw-
mask::rw-
other::---
```
Please note the line `user:kodemeister:rw-`. This means that user `kodemeister` has R/W access to `/dev/uinput` in addition to standard Unix user/group/other permissions.
3. We run Docker container and mount `/dev` directory from host to container (`-v /dev:/dev`).
4. We create a new `lizard` user in the container and assign it the same UID as `kodemeister` user on the host.
5. As a result, `lizard` user should have access to `/dev/uinput` without messing with group permissions.

Indeed, this needs to be tested. Are there any instructions on running dockerized Sunshine? Setting up GUI apps in a container is a bit of pain.

Reply:
Thanks, makes perfect sense to me.
I'm on my phone, but I'll try just to add a couple of random notes:
- There's no need to pass the full `/dev` to the docker container you can add: `-v /dev/input:/dev/input:rw` and `--device /dev/uinput` and that should be enough
- As for running Sunshine in Docker I'm the main mantainer of Games on Whales https://github.com/games-on-whales/gow where we build a little image on top of Sunshine (coincidentally it's  mainly just to setup additional user/group access) and a bunch of others so that you can "easily" run this in Docker.